src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/dto/SessionTemplateDto.kt

```diff
@@ -33,9 +33,7 @@ data class SessionTemplateDto(
   @Schema(description = "open capacity", example = "50", required = true)
   val openCapacity: Int,
   @Schema(description = "day of week fpr visit", example = "MONDAY", required = false)
-  val dayOfWeek: DayOfWeek?,
-  @Schema(description = "List of prison wings associated with the session template, if list is empty then all wings are acceptable!", required = false)
-  val prisonWings: List<SessionPrisonWingDto> = mutableListOf()
+  val dayOfWeek: DayOfWeek?
```

Comment:
sorry not to do with the current change, just noticed this. Should dayOfWeek here be not null as it cannot be null on DB.